.github/workflows/cve_scan_runner.yml

```
@@ -0,0 +1,36 @@
+name: cve-scan-runner
+
+on:
+  workflow_dispatch:
+
+  schedule:
+    - cron: "0 0 * * *"
```

```suggestion
    - cron: "17 23 * * *"
```

Practically speaking, this tends to be a busy time for cronjobs. Consider something not on the 0s.